packages/shared/src/components/search/SearchBarSuggestionList.tsx

```diff
@@ -3,30 +3,46 @@ import classNames from 'classnames';
 import { SearchBarSuggestion } from './SearchBarSuggestion';
 import AuthContext from '../../contexts/AuthContext';
 import FeedbackIcon from '../icons/Feedback';
-import { getSearchIdUrl, SearchSession } from '../../graphql/search';
+import { getSearchUrl, SearchSession } from '../../graphql/search';
+import classed from '../../lib/classed';
+import { ElementPlaceholder } from '../ElementPlaceholder';
 
 interface SearchBarSuggestionListProps {
   className?: string;
-  suggestions?: SearchSession[];
+  isLoading?: boolean;
+  suggestions?: Partial<SearchSession>[];
 }
 
+const Pill = classed(ElementPlaceholder, 'h-8 rounded-12 w-auto');
```

Comment:
You defined it below as utils, can't you use those?

Reply:
Oh, copy-paste issue. Let me move this one.